### PR TITLE
FAQ: revise entry on Scala versionNumberString

### PR DESCRIPTION
**Pull request by SethTisue**
review by @ckipp01 ?

### CODE REVIEW

**Comment by SethTisue** [comment]
The CI failure is unrelated to this change. A different open PR has a fix.

File: _overviews/FAQ/index.md [hunk 1]
```diff
@@ -108,20 +108,25 @@ Your sbt 1.x build definition is always a Scala 2.12 program.
...
 
-### I want Scala 3.1.1 (etc); why does std lib say it's using Scala 2.13?
+### I want Scala 3. Why does `versionNumberString` say I'm on 2.13?
 
-Scala 3 currently uses the Scala 2.13 library by leveraging its seamless
-interoperability. Note that it does not necessarily ingest the latest
-version of the Scala 2.13 library.
+To aid migration, Scala 3 currently uses the Scala 2.13 library as-is.
+That's why `versionString` and `versionNumberString` give a
+perhaps-unexpected Scala 2 answer:
```
Comment (som-snytt) [inline _overviews/FAQ/index.md:115]: I wonder what would be the German word for "a perhaps-unexpected Scala 2 answer".

"That's why [they] report a Scala 2 version."

File: _overviews/FAQ/index.md [hunk 2]
```diff
@@ -108,20 +108,25 @@ Your sbt 1.x build definition is always a Scala 2.12 program.
 Regardless, in your `build.sbt`, you can set `scalaVersion` to whichever
 available distribution you want and your program code will be compiled with that version.
 
-### I want Scala 3.1.1 (etc); why does std lib say it's using Scala 2.13?
+### I want Scala 3. Why does `versionNumberString` say I'm on 2.13?
 
-Scala 3 currently uses the Scala 2.13 library by leveraging its seamless
-interoperability. Note that it does not necessarily ingest the latest
-version of the Scala 2.13 library.
+To aid migration, Scala 3 currently uses the Scala 2.13 library as-is.
```
Comment (som-snytt) [inline _overviews/FAQ/index.md:113]: According to all the crossword puzzle clues, "AS IS" is the answer to, "Common classified clause" or "Flea-market condition", i.e., "Won't fix."

"...library, supplemented by a few bits that are specific to Scala 3."

**Review by som-snytt — CHANGES_REQUESTED** [review]
English is such a hard language.

**Review by som-snytt — APPROVED** [review]
Crystal!

**Comment by SethTisue** [comment]
(I did tweak it in response to the review feedback before merge.)

**Pull request merged**